javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,31 @@
+function weatherWear(temperature){
+    let clothesToWear ={
```

Comment:
Cool use of object and arrays! 
 Be aware that item indexes of an array might change. So unless the items and their order are a constant, you might run into trouble further down the line.

Reply:
So I changed it to const